diffpy/snmf/tests/test_subroutines.py

```diff
@@ -134,3 +134,20 @@ def test_reconstruct_data(trd):
     actual = reconstruct_data(trd[0][0], trd[0][1], trd[0][2], trd[0][3], trd[0][4], trd[0][5])
     expected = trd[1]
     np.testing.assert_allclose(actual, expected, rtol=1e-03)
+
+
+tld = [([[[1, -1, 1], [0, 0, 0], [2, 10, -3]], 1], [[4, 2, 4], [3, 3, 3], [5, 13, 0]]),
```

Comment:
to make these easier to read, maybe make the inputs and outputs as tuples, something like:
```
tld = [
          (([[1, -1, 1], [0, 0, 0], [2, 10, -3]], 1), 
               ([4, 2, 4], [3, 3, 3], [5, 13, 0])),
          (([[1, -1, 1], [0, 0, 0], [2, 10, -3]], 0), 
                ([1, -1, 1], [0, 0, 0], [2, 10, -3])),
```
and so on.

Reply:
I will do this